engine/src/funcs.cpp

```diff
@@ -578,6 +578,8 @@ void MCChunkOffset::compile(MCSyntaxFactoryRef ctxt)
 	case CT_CHARACTER:
 		MCSyntaxFactoryEvalMethod(ctxt, kMCStringsEvalOffsetMethodInfo);
 		break;
+    default:
+        break;
```

Comment:
I'm pretty certain this should be `MCUnreachable()`. If reached it would indicate that the parser had accepted something that couldn't actually be compiled.